bot/code_review_bot/report/base.py

```diff
@@ -30,6 +31,7 @@
         "defect": " - {nb} found by mozlint",
         "analyzer": " - `./mach lint --warnings path/to/file` (JS/Python/etc)",
     },
+    DefaultIssue: {"defect": " - {nb} found by a generic analyzer"},
```

Comment:
Could we use the name of the analyzer here?

Reply:
In a followup yes (under #233), as it involves modifying the stats calculation and how we refer to analyzers.